src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -141,7 +143,7 @@ public void check_mail_subscriptions_were_triggered() throws Throwable {
         }
     }
     
-    @Then("^Rest subscriptions were triggered$")
+    @And("^Rest subscriptions were triggered$")
     public void check_rest_subscriptions_were_triggered() throws Throwable {
         LOGGER.debug("Verifying REST requests.");
         assert(requestBodyContainsStatedValues(new JSONArray(mockClient.retrieveRecordedRequests(request().withPath(REST_ENDPOINT), Format.JSON))));
```

Comment:
Here it would be good to compare actual AggrObject with expected AggrObject,, so we easy can see what have not aggregated when trigger fails to trigger.

Reply:
We don't have a prepared object but we do check that expected events and parameters are present atm.